lib/services/plugins-service.ts

```diff
@@ -257,5 +239,18 @@ export class PluginsService implements IPluginsService {
 
 		return plugin;
 	}
+
+	private getAllInstalledPlugins(configuration: {getPluginsWithoutOptions: (...args: any[]) => any[]}): IPlugin[] {
```

Comment:
yes, I was confused of the name as well, operation seems better for me :) 
btw great idea to pass the operation instead of boolean value